config/smartyadmin.config.inc.php

```diff
@@ -47,6 +47,9 @@
 smartyRegisterFunction($smarty, 'block', 'addJsDefL', array('Media', 'addJsDefL'));
 smartyRegisterFunction($smarty, 'modifier', 'secureReferrer', array('Tools', 'secureReferrer'));
 
+$module_resources['modules'] = _PS_MODULE_DIR_;
+$smarty->registerResource('module', new SmartyResourceModule($module_resources, $isAdmin = true));
```

Comment:
Careful, you are assigning `true` to $isAdmin. Maybe you just wanted to send true ?